sw/host/tests/chip/sysrst_ctrl/sysrst_ctrl_ec_rst_l.rs

```diff
@@ -0,0 +1,232 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{bail, ensure, Result};
+use clap::Parser;
+use once_cell::sync::Lazy;
+use std::borrow::Borrow;
+use std::collections::HashMap;
+use std::rc::Rc;
+use std::time::Duration;
+
+use opentitanlib::app::TransportWrapper;
+use opentitanlib::io::gpio::{ClockNature, Edge, MonitoringEvent, PinMode};
+use opentitanlib::io::uart::Uart;
+use opentitanlib::test_utils::init::InitializeTest;
+use opentitanlib::test_utils::test_status::TestStatus;
+use opentitanlib::uart::console::UartConsole;
+use opentitanlib::{collection, execute_test};
+
+use sysrst_ctrl::{Config, setup_pins, set_pins, read_pins};
+
+#[derive(Debug, Parser)]
+struct Opts {
+    #[command(flatten)]
+    init: InitializeTest,
+
+    /// Console receive timeout.
+    #[arg(long, value_parser = humantime::parse_duration, default_value = "10s")]
+    timeout: Duration,
+}
+
+struct Params<'a> {
+    opts: &'a Opts,
+    transport: &'a TransportWrapper,
+    uart: &'a dyn Uart,
+    config: &'a Config,
+}
+
+static CONFIG: Lazy<HashMap<&'static str, Config>> = Lazy::new(|| {
+    collection! {
+        "hyper310" => Config {
+            // key0_in, key1_in
+            output_pins: vec!["IOR6", "IOR7"],
+            open_drain: vec![false, false],
+            // ec_rst, flash_wp
+            input_pins: vec!["IOR8", "IOR9"],
+            // ec_rst, flash_wp
+            pullup_pins: vec!["IOR8", "IOR9"],
+        },
+    }
+});
+
+const COMBO_PINS_HIGH: u8 = 0x3;
+const COMBO_PINS_LOW: u8 = 0;
+const COMBO_PINS_KEY0_HIGH: u8 = 1;
+const EC_RST_FLASH_WP_HIGH: u8 = 0x3;
+const EC_RST_PIN_INDEX: u8 = 0;
+const FLASH_WP_PIN_INDEX: u8 = 1;
+const EC_RST_PIN: &str = "IOR8";
+
+const EC_RST_COMBO_PULSE_WIDTH_US: u64 = 100_000; // Set on the device.
```

Comment:
I've seemed `micros` rather than `us` as sufix in Opentitan tests.

Reply:
I looked at the code and it's a bit random. I will change it to `USEC` since it's used in another rust test and in several device tests.